_includes/nav.html

```diff
@@ -1,64 +1,62 @@
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown">Community</a>
           <ul class="dropdown-menu">
-            <li><a href="{{ site.baseurl }}/contact-us">Contact Us</a></li>
-            <li><a href="{{ site.baseurl }}/how-to-contribute">How To Contribute</a></li>
-            <li><a href="{{ site.baseurl }}/people">People</a></li>
-            <li><a href="{{ site.baseurl }}/related-projects">Related Projects</a></li>
+            <li><a class="dropdown-item" href="{{ site.baseurl }}/contact-us">Contact Us</a></li>
+            <li><a class="dropdown-item" href="{{ site.baseurl }}/how-to-contribute">How To Contribute</a></li>
+            <li><a class="dropdown-item" href="{{ site.baseurl }}/people">People</a></li>
+            <li><a class="dropdown-item" href="{{ site.baseurl }}/related-projects">Related Projects</a></li>
           </ul>
         </li>
-        <li class="nav-link"><a href="{{ site.baseurl }}/search">Search</a></li>
+        <li class="nav-item"><a class="nav-link" href="{{ site.baseurl }}/search">Search</a></li>
       </ul>
-      <ul class="nav navbar-nav navbar-right">
-        <li class="dropdown">
-          <a class="dropdown-toggle" data-toggle="dropdown" href="#"><img alt="Apache Software Foundation" src="https://www.apache.org/foundation/press/kit/feather.svg" width="15"/><span class="caret"></span></a>
+      <ul class="navbar-nav ms-auto">
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown">
+            <img alt="Apache Software Foundation" src="https://www.apache.org/foundation/press/kit/feather.svg" width="15"/>
+          </a>
           <ul class="dropdown-menu">
-            <li><a href="https://www.apache.org">Apache Homepage <i class="fa-solid fa-up-right-from-square"></i></a></li>
-            <li><a href="https://www.apache.org/licenses/">License <i class="fa-solid fa-up-right-from-square"></i></a></li>
-            <li><a href="https://www.apache.org/foundation/sponsorship">Sponsorship <i class="fa-solid fa-up-right-from-square"></i></a></li>
-            <li><a href="https://www.apache.org/security">Security <i class="fa-solid fa-up-right-from-square"></i></a></li>
-            <li><a href="https://www.apache.org/foundation/thanks">Thanks <i class="fa-solid fa-up-right-from-square"></i></a></li>
-            <li><a href="https://www.apache.org/foundation/policies/conduct">Code of Conduct <i class="fa-solid fa-up-right-from-square"></i></a></li>
-            <li><a href="https://www.apache.org/events/current-event.html">Current Event <i class="fa-solid fa-up-right-from-square"></i></a></li>
+            <li><a class="dropdown-item" href="https://www.apache.org">Apache Homepage <i class="fa-solid fa-up-right-from-square"></i></a></li>
```

Comment:
Fixed in 6dd2577

Reply:
I looked at the documentation, and I'm not sure I'm right about this. I framed it more as a question, because I wasn't sure. The documentation seems to indicate it's primarily for the keyboard navigation, not part of the visual of the menu. I'm still not sure why it's needed explicitly, instead of being inferred, but I don't have much free time to play with it.

I would follow the bootstrap examples, though. The correct answer to my previous questions probably should have been "no". Sorry if I created more churn by asking this.

Reply:
I'll take a second look and change things back if needed. Everything seems to be working okay as is though.

Reply:
Yea it looks like convention is to have the class within the `<a>` so I moved it back in 239694b